Tests/test_AlignInfo.py

```diff
@@ -139,7 +139,7 @@ def test_pseudo_count(self):
         ic = summary.information_content(e_freq_table=expected,
                                          log_base=math.exp(1),
                                          pseudo_count=1)
-        ic_vector = summary.ic_vector.values()
+        ic_vector = list(summary.ic_vector.values())
```

Comment:
See commit 09448850. The keys of  summary.ic_vector are the columns number.  So 

```python
 ic_vector = [summary.ic_vector[i]  for (i, k) in enumerate(summary.ic_vector)]
```
return the vector of information content in the correct order.

Yes, it apparently won't break any part of the code if it is changed to a list.